configs/dlstreamer/framework-pipelines/yolov5_pipeline/yolov5s.sh

```diff
@@ -12,4 +12,4 @@ AGGREGATE="gvametaaggregate name=aggregate !" # Aggregate function at the end of
 OUTPUTFORMAT="! fpsdisplaysink video-sink=fakesink sync=true --verbose" # Output format for the pipeline  "! fpsdisplaysink video-sink=fakesink sync=true --verbose" | (render_mode)"! videoconvert ! video/x-raw,format=I420 ! gvawatermark ! videoconvert ! fpsdisplaysink video-sink=ximagesink sync=true --verbose"
 
 echo "Run yolov5s pipeline"
-gst-launch-1.0 $inputsrc ! $DECODE ! gvadetect model-instance-id=odmodel name=detection model=models/yolov5s/1/FP32-INT8/yolov5s.xml model-proc=models/yolov5s/1/yolov5s.json threshold=.5 device=$DEVICE $PRE_PROCESS ! $AGGREGATE gvametaconvert name=metaconvert add-empty-results=true ! gvametapublish name=destination file-format=2 file-path=/tmp/results/r$cid_count.jsonl $OUTPUTFORMAT 2>&1 | tee >/tmp/results/gst-launch_$cid_count.log >(stdbuf -oL sed -n -e 's/^.*current: //p' | stdbuf -oL cut -d , -f 1 > /tmp/results/pipeline$cid_count.log)
\ No newline at end of file
+gst-launch-1.0 $inputsrc ! $DECODE ! gvadetect model-instance-id=odmodel name=detection model=models/yolov5s/FP32-INT8/1/yolov5s.xml model-proc=models/yolov5s/FP32-INT8/1/yolov5s.json threshold=.5 device=$DEVICE $PRE_PROCESS ! $AGGREGATE gvametaconvert name=metaconvert add-empty-results=true ! gvametapublish name=destination file-format=2 file-path=/tmp/results/r$cid_count.jsonl $OUTPUTFORMAT 2>&1 | tee >/tmp/results/gst-launch_$cid_count.log >(stdbuf -oL sed -n -e 's/^.*current: //p' | stdbuf -oL cut -d , -f 1 > /tmp/results/pipeline$cid_count.log)
```

Comment:
Note: with the directory structure change we'll need to download the proc.json for each precision of a model. This is currently only used for DLStreamer pipelines so it as minimal impact for now.

Reply:
Where do we grab the proc.json for each model and is this something we want to implement now?